package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "Js project Restaurant Page",
+  "main": "stylelint.config.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
- [ ] [REQUIRED] It should include the necessary commands to build in the package.json
![image](https://user-images.githubusercontent.com/73254040/101965860-ae1dd780-3bdb-11eb-9b0f-d05fac3b97ff.png)
[Source](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)